### PR TITLE
Pseudocode compiler

### PR DESCRIPTION
**Pull request by emergenitro**
I think I've completed #105 . Let me know if there are any errors/problems.

Thanks!

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -2174,5 +2174,36 @@ async def apply(interaction: discord.Interaction):
...
+        self.code = discord.ui.TextInput(
+            label = "Code",
+            style = discord.TextInputStyle.full,
+            placeholder = "The code you would like to parse",
+            required = True
+        )
+        self.add_item(self.code)
+    
+    async def callback(self, interaction: discord.Interaction):
+        print(self.code.value)
```
Comment (kr1zz7007) [inline app.py:2190]: can you remove the print function here?

Comment (emergenitro) [reply app.py:2190]: oh right yes my bad

File: app.py [hunk 2]
```diff
@@ -2174,5 +2174,36 @@ async def apply(interaction: discord.Interaction):
     view.add_item(ApplyDropdown())
     await interaction.send(view = view, ephemeral = True)
 
+class Code(discord.ui.Modal):
+    def __init__(self):
+        super().__init__("Code", timeout = None)
+
+        self.code = discord.ui.TextInput(
+            label = "Code",
+            style = discord.TextInputStyle.full,
```
Comment (kr1zz7007) [inline app.py:2183]: I think this would be `discord.TextInputStyle.paragraph`

**Comment by emergenitro** [comment]
Apologies for the constant changes in the wording

**Review by Sachin-dot-py — APPROVED** [review]

**Pull request merged**